XamarinCommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+    public class UserStoppedTypingBehavior : BaseBehavior
+    {
+        CancellationTokenSource tokenSource;
+
+        #region Bindable Properties
+
+        public static readonly BindableProperty CommandProperty = BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(UserStoppedTypingBehavior));
+        public ICommand Command
+        {
+            get => (ICommand)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        public static readonly BindableProperty StoppedTypingThresholdProperty = BindableProperty.Create(nameof(StoppedTypingThreshold), typeof(int), typeof(UserStoppedTypingBehavior), defaultValue: 1000);
+        public int StoppedTypingThreshold
```

Comment:
I think it makes sense to clarify what this means.
StoppedTypingTimeThreshold or StoppedTypingMillisecondsThreshold